Dockerfile

```diff
@@ -97,6 +98,15 @@ ENV PATH="${PATH}:/app/.eval-dev-quality/go/bin"
 ENV GOROOT="/app/.eval-dev-quality/go"
 ENV PATH="${PATH}:/home/ubuntu/go/bin"
 
+# Install Rust
+ENV RUSTUP_HOME="/app/.eval-dev-quality/rustup"
+ENV CARGO_HOME="/app/.eval-dev-quality/cargo"
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --no-modify-path && \
+	/app/.eval-dev-quality/cargo/bin/rustup install 1.86.0 && \
+	/app/.eval-dev-quality/cargo/bin/rustup component add llvm-tools-preview && \
+	/app/.eval-dev-quality/cargo/bin/cargo install cargo-llvm-cov
```

Comment:
why chaining all of this commands? why not do separate "RUN"s?

please use the env variable in the paths: `${CARGO_HOME}/bin/rustup` or add cargo/bin to the path before

Reply:
I am no expert with Docker but AFAIK for every `RUN` command, docker takes a separate "snapshot" of the file system. So it is common to group multi-command operations under a single `RUN` (I think there are some other cases in our Dockerfile).

https://docs.docker.com/reference/dockerfile/#run